src/css.js

```diff
@@ -266,6 +266,11 @@ jQuery.extend( {
 
 				if ( isCustomProp ) {
 					style.setProperty( name, value );
+				} else if ( value.indexOf( "!important" !== -1 ) ) {
```

Comment:
Custom properties' values may have the `!important` clause as well, this needs to be handled as well.

Reply:
Right. Unit tests would immediately catch this mistake. :)